.circleci/config.yml

```diff
@@ -797,19 +799,22 @@ jobs:
           command: |
             set -x
             BRANCH=${CIRCLE_BRANCH}
-            docker/build_push.sh -u -b ${BRANCH} -n client
-            docker/build_push.sh -u -b ${BRANCH} -n init
-            docker/build_push.sh -u -b ${BRANCH} -n faucet
-            docker/build_push.sh -u -b ${BRANCH} -n tools
-            docker/build_push.sh -u -b ${BRANCH} -n validator
-            docker/build_push.sh -u -b ${BRANCH} -n validator-tcb
-            docker/build_push.sh -u -b ${BRANCH} -n cluster-test
+            success=0
+            docker/build_push.sh -u -b ${BRANCH} -n client || success=-1
+            docker/build_push.sh -u -b ${BRANCH} -n init || success=-1
+            docker/build_push.sh -u -b ${BRANCH} -n faucet || success=-1
+            docker/build_push.sh -u -b ${BRANCH} -n tools || success=-1
+            docker/build_push.sh -u -b ${BRANCH} -n validator || success=-1
+            docker/build_push.sh -u -b ${BRANCH} -n validator-tcb || success=-1
+            docker/build_push.sh -u -b ${BRANCH} -n cluster-test || success=-1
+            exit $success
       - run:
           name: docker image pruning.
           command: |
             scripts/dockerhub_prune.sh -u "${DOCKERHUB_USERNAME}" -p "${DOCKERHUB_PASSWORD}" -x
       - run:
           name: push to novi ecr
+          when: always
```

Comment:
consider this says always... why do we need the rest?

Reply:
We want to always attempt the push to Novi ECR; it might fail (if say the Rust build is failing), but if the issue is just with Dockerhub it at least gets us images in ECR.

Reply:
Comment was more that the PR title and code ... aren't quite aligned... I agree on the approach taken.

Reply:
the prior steps build the images, this merely pushes them to novi aws ecr.